### PR TITLE
[201911] [Flex counters] Add SAI_INGRESS_PRIORITY_GROUP_STAT_DROPPED_PACKETS counter, create new FlexCounter group

### PR DESCRIPTION
**Pull request by ayurkiv-nvda**
Appropriate PR in master https://github.com/Azure/sonic-swss/pull/1600

**What I did**
Added possibility to calculate Priority Group drop packet counters 

**Why I did it**
Need to merge feature from master to 201911

**How I verified it**
1) Setup a testbed like this: hostA - DUT – hostB, layer 2 or 3 forwarding between A/B. better to use layer 3 because by using layer 3 we can easily designate DSCP of the packet.
Make sure all entries for forwarding have been correctly established, ARP, routing, neighbor, …
2) Assume you will send traffic from A to B, start PFC storm on host B. if you are using SONiC testbed, you can start PFC storm on the fanout switch. (you can consult Volodymyr regarding how to start PFC storm)
3) Check and update the buffer profile on the DUT’s ports connected to host A:
Let’s say it’s Ethernet8, using the following command:
```
admin@arc-mtbc-1001:~$ redis-cli -n 4 hgetall "BUFFER_PG|Ethernet8|3-4"
1) "profile"
2) "[BUFFER_PROFILE|pg_lossless_25000_5m_profile]"
admin@arc-mtbc-1001:~$ redis-cli -n 4 hset "BUFFER_PROFILE|pg_lossless_25000_5m_profile" dynamic_th -8
```

and then save config and reload config.
    Send packets from host A to host B, make sure the DSCP is 3.
You should be able to observe discard on PG 3 soon.

**Details if related**

### CODE REVIEW

**Comment by prsunny** [comment]
Can you provide a link to master PR?

**Comment by ayurkiv-nvda** [comment]
> Can you provide a link to master PR?



> Can you provide a link to master PR?

@prsunny https://github.com/Azure/sonic-swss/pull/1600

**Comment by liat-grozovik** [comment]
@neethajohn following your ask enclose in the backport PR. Appreciate if you can prioritized the review.

File: tests/test_pg_drop_counter.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        self.pgs = self.get_oids(dvs, "SAI_OBJECT_TYPE_INGRESS_PRIORITY_GROUP")
+        self.set_up_flex_counter(dvs)
+
+        self.populate_asic(dvs, "100")
+        time.sleep(self.DEFAULT_POLL_INTERVAL + 1)
+        self.verify_value(dvs, self.pgs, pg_drop_attr , "100")
+
+        self.populate_asic(dvs, "123")
+        time.sleep(self.DEFAULT_POLL_INTERVAL + 1)
+        self.verify_value(dvs, self.pgs, pg_drop_attr, "123")
```
Comment (neethajohn) [inline tests/test_pg_drop_counter.py:97]: please put this in try-finally and clean up the flex counter

File: tests/test_pg_drop_counter.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    def set_up_flex_counter(self, dvs):
+        for pg in self.pgs:
+            dvs.runcmd("redis-cli -n 5 hset 'FLEX_COUNTER_TABLE:PG_DROP_STAT_COUNTER:{}' ".format(pg) + "PG_COUNTER_ID_LIST '{}'".format(pg_drop_attr))
+
+        dvs.runcmd("redis-cli -n 4 hset 'FLEX_COUNTER_TABLE|PG_DROP' 'FLEX_COUNTER_STATUS' 'enable'")
+
+        self.populate_asic(dvs, "0")
+
+
+    def set_up(self, dvs):
```
Comment (neethajohn) [inline tests/test_pg_drop_counter.py:70]: what is this function for?

File: tests/test_pg_drop_counter.py (hunk 1 above)
Comment (ayurkiv-nvda) [reply tests/test_pg_drop_counter.py]: Done

File: tests/test_pg_drop_counter.py (hunk 2 above)
Comment (ayurkiv-nvda) [reply tests/test_pg_drop_counter.py]: removed

**Comment by liat-grozovik** [comment]
@ayurkiv-nvda could you please followup on the LGTM issue raised in this PR? if this infra issue please contact the build team.

**Comment by liat-grozovik** [comment]
@ayurkiv-nvda kindly reminder to check the LGTM

**Comment by ayurkiv-nvda** [comment]
> @ayurkiv-nvda kindly reminder to check the LGTM

@liat-grozovik It looks like these LGTM issue for 201911 is know problem
There are PR's that were merged despite the LGTM test failure:  https://github.com/Azure/sonic-swss/pull/1901, https://github.com/Azure/sonic-swss/pull/2196, https://github.com/Azure/sonic-swss/pull/2279
I think we can ignore it

**Comment by ayurkiv-nvda** [comment]
@liat-grozovik 
hello
so, can we merge this PR despite LGTM failure It looks safe?

**Pull request merged**